array-methods/for-each.js

```diff
@@ -0,0 +1,7 @@
+const values = [10, 1, 22, 23, 41, 5, 18, 7, 80, 9];
+values.forEach((element) => console.log(element));
+values.forEach((element) => {
+  const index = values.indexOf(element);
```

Comment:
Instead of finding `index` this way, you should be utilizing your callback function and adding another parameter as your `index`.

The problem with your current solution is that `indexOf` under the hood is doing another loop. So now you're doing a nested loop.